poetry/poetry.py

```diff
@@ -77,15 +74,15 @@ def create(cls, cwd):  # type: (Path) -> Poetry
 
         else:
             raise RuntimeError(
-                "Poetry could not find a pyproject.toml file in {} or its parents".format(
-                    cwd
-                )
+                "Poetry could not find a pyproject.toml \
+file in {} or its parents".format(cwd)
```

Comment:
And this looks bad too.

Reply:
is this better?
```

            raise RuntimeError(
                "Poetry could not find a pyproject.toml "
                "file in {} or its parents".format(cwd)
            )
```
I think it would be nice to have some indication what would be acceptable so we don't have to guess in case where linter says its ok

Reply:
Personally i like it, nice improvement.

Reply:
paint all black!